sdks/java/io/cdap/src/main/java/org/apache/beam/sdk/io/cdap/context/BatchSinkContextImpl.java

```diff
@@ -24,7 +24,10 @@
 public class BatchSinkContextImpl extends BatchContextImpl implements BatchSinkContext {
 
   @Override
-  public void addOutput(Output output) {}
+  public void addOutput(Output output) {
```

Comment:
this probably should be setoutput if it can only be one element

if it is addoutput, it should be some form of a collection of elements

Reply:
This class implements the CDAP (io.cdap.cdap.etl.api.batch) interface, so we cannot change the method name.

Reply:
Hmm. its probably worth adding a comment here to help mitigate future confusion

Reply:
Please add the comment here